### DISCUSSION TITLE
UNION ALL operator in postgresql

### DISCUSSION THREAD

**Discussion by sherif-927** [toluaina/pgsync#248] [Q&A]
Right now I have a bunch of PostgreSQL Views that I need to re-write into elasticsearch indexes using pgsync , however I came across a particular problem, which is that I cannot find anything to emulate the "UNION ALL" query in postgresql , so for example if I have this view 
![UNION_ALL](https://user-images.githubusercontent.com/69459783/153717623-5c5ee83a-968a-4acf-a46e-7720af802c6f.png)
Is there anyway for me to achieve a similar result when using pgsync ?

**Comment by toluaina** [comment]
Hello and sorry for the delay. I have only recently been looking at the discussion section.
There is no way to achieve this using pgsync as its a very specific case. 
I'm planning to take a look at view soon so this might help

**Reply by sherif-927** [reply]
@toluaina are there any future plans for generating the elastic search indexes from user-defined SQL queries in postgresql dialect instead of the schema files ? like AWS Glue Elastic Views for instance

**Reply by toluaina** [reply]
Hmm. there are no plans for this but a POC might be a good place to start. Although I don't  have the capacity to take this on at the moment
